granary/meetup.py

```diff
@@ -99,3 +100,36 @@ def _create(self, obj, preview=False, include_link=source.OMIT_LINK, ignore_form
 
     def return_error(self, msg):
         return source.creation_result(abort=True, error_plain=msg, error_html=msg)
+
+    def user_to_actor(self, user):
+      """Converts a user to an actor.
+
+      Args:
+        user: dict, a decoded JSON Meetup user
+
+      Returns:
+        an ActivityStreams actor dict, ready to be JSON-encoded
+      """
+      user_id = user.get('id')
+      user_id_str = str(user_id),
```

Comment:
you'll want to remove the trailing comma so this is a string and not a tuple. silly little python quirk 😜

Reply:
Haha woops, that was a bad copy-paste from me